samples/python/image_generation/README.md

```diff
@@ -36,6 +36,19 @@ pip install --upgrade-strategy eager -r ../../export-requirements.txt
 optimum-cli export openvino --model dreamlike-art/dreamlike-anime-1.0 --task stable-diffusion --weight-format fp16 dreamlike_anime_1_0_ov/FP16
 ```
 
+Alternatively, do it in Python code:
```

Comment:
This one is not alternative for requirements installation. Please split the export with the requirements installation